templates/index.html

```diff
@@ -145,12 +145,7 @@ <h3 class="section-head">Live Stream</h3>
                             <h3 class="section-head">Real Time Analysis</h3>
                             <div class="rev-content">
                                 <img src="./youtubelogo.jpg" alt="">
-                                <div class="rev-info">
-                                    <p>10:04:30 The patient is happy with confidence 74% </p>
-                                    <p>10:04:50 The patient is happy with confidence 96% </p>
-                                    <p>10:05:23 The patient is sad with confidence 30% </p>
-                                    <p>10:05:47 The patient is happy with confidence 55% </p>
-                                    <p>10:06:10 The patient is happy with confidence 88% </p>
+                                <div class="rev-info emotion_state" id="log_section">
```

Comment:
Please remove `emotion_state` class and use `log_section` id to refer to this object, since there are no other elements with `emotion_state` class.